src/ts3init_target.c

```diff
@@ -460,7 +460,7 @@ ts3init_get_cookie_ipv4_tg(struct sk_buff *skb, const struct xt_action_param *pa
     if (skb->len > dst_mtu(skb_dst(skb)))
         return NF_DROP;
 
-    return NF_CONTINUE;
+    return XT_CONTINUE;
```

Comment:
These were changed from NF_ACCEPT to XT_CONTINUE. This would change the way these matches work, since right now it is expected that a packet stops traversing the chain once it matches this rule. Is there a good reason to do this?